crates/client/sync/src/eth/src/config.rs

```diff
@@ -0,0 +1,143 @@
+//! Base Ethereum client configuration.
+//!
+//! Use it as is or reuse top-level fields in your config ("inherit")
+//! in order to share the same configuration file.
+//!
+//! struct MyServiceConfig {
+//!     pub provider: EthereumProviderConfig,
+//!     pub wallet: Option<EthereumWalletConfig>,
+//!     pub contracts: Option<StarknetContracts>,
+//! }
+//!
+//! Default provider and wallet configurations are set for use with Anvil
+//! - local Ethereum environment.
+
+use std::fs::File;
+use std::path::PathBuf;
+use alloy::network::EthereumWallet;
+use alloy::primitives::{
+    Address
+};
+use serde::{Deserialize, Serialize};
+
+use crate::error::Error;
+// use crate::oracle::OracleConfig;
+
+/// Default Anvil local endpoint
+pub const DEFAULT_RPC_ENDPOINT: &str = "http://127.0.0.1:8545";
+/// Default Anvil chain ID
+pub const DEFAULT_CHAIN_ID: u64 = 31337;
+/// Default private key derived from starting Anvil as follows:
+/// anvil -b 5 --config-out $BUILD_DIR/anvil.json
+/// PRE_PRIVATE=$(jq -r '.private_keys[0]' $BUILD_DIR/anvil.json)
+pub const DEFAULT_PRIVATE_KEY: &str = "ac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80";
+
+#[derive(Debug, Clone, Default, Serialize, Deserialize)]
+pub struct EthereumClientConfig {
+    #[serde(default)]
+    pub provider: EthereumProviderConfig,
+    #[serde(default, skip_serializing_if = "Option::is_none")]
+    pub wallet: Option<EthereumWalletConfig>,
```

Comment:
we can remove the wallet right? we won't be sending any txs from the sequencer

Reply:
yes removed